custom/icds_reports/templates/icds_reports/icds_app/mobile/partials/mobile-heading.html

```diff
@@ -11,11 +11,12 @@
                 {% endcomment %}
                 {$ $ctrl.selectedMonthDisplay $} |
                 <span ng-click="$ctrl.getDisableIndex() < 0 && $ctrl.moveToLocation('national', -1)"
-                      ng-class="{'d-none':($ctrl.selectedLocationsCount()>0)}">National</span>
+                      ng-class="{'d-none':($ctrl.selectedLocationsCount()>0), 'clickable-breadcrumb': ($ctrl.selectedLocationsCount()==0)}">National</span>
                 <span ng-click="$ctrl.getDisableIndex() < $index && $ctrl.moveToLocation(loc, $index)"
                       ng-class="{'d-none':($index != $ctrl.selectedLocationsCount()) && ($index != $ctrl.selectedLocationsCount()-1)}"
                       ng-repeat="loc in $ctrl.selectedLocations | filter: { location_id: '!all'} track by $index">
-                  <span ng-show="($ctrl.selectedLocationsCount()==$index)">&gt;</span> {$ loc.name $}
+                  <span ng-show="($ctrl.selectedLocationsCount()==$index)">&gt;</span>
+                  <span ng-class="{'clickable-breadcrumb':(($ctrl.selectedLocationsCount()-1)==$index)}">{$ loc.name $}</span>
```

Comment:
not a big deal, though feel free to follow up if you want - it's a bit odd that the styling is applied in here, but the click target actually includes the `>` character. I think it might make sense for the click target to just be the name.

Reply:
Also do you need any other conditions? e.g. `$ctrl.getDisableIndex()`?

Reply:
I will move ng-click from parent span element to just the location name